daemon/graphdriver/lcow/lcow.go

```diff
@@ -219,21 +219,21 @@ func (d *Driver) startServiceVMIfNotRunning(id string, mvdToAdd []hcsshim.Mapped
 	// Use the global ID if in global mode
 	id = d.getVMID(id)
 
-	title := fmt.Sprintf("lcowdriver: startservicevmifnotrunning %s:", id)
+	title := "lcowdriver: startServiceVMIfNotRunning " + id
```

Comment:
Perhaps for a follow up, but we should `logrus.WithFields` for this information (similar to what we do for other storage drivers)

Reply:
I concur (see the patch description).

Reply:
...but I'd rather do it in a separate PR later as this fix is quite big already -- and initially it supposed to be a one-liner :rofl:

Reply:
D'oh!! Didn't read that far into your description 🤦‍♂️☺️😝